ConceptMap/ConceptMap-England-ConditionTumourStatus.json

```diff
@@ -43,8 +42,8 @@
           "display": "PRIMARY",
           "target": [
             {
-              "code": "128462008",
-              "display": "Metastatic malignant neoplasm (disorder)",
+              "code": "63161005",
+              "display": "Primary",
```

Comment:
As you've used the FSN for the other display names, could you replace Primary with: Principal (qualifier value)